app/routes/account.py

```diff
@@ -146,3 +146,11 @@ def register_user():
         return json_api(e, ErrorSchema), 422
     except ValidationError as e:
         return json_api(BadRequest(source=e.message), ErrorSchema), 400
+
+
+@account_blueprint.route("/account", methods=["GET"])
+@jwt_required
+def get_user():
+    """ Delete an existing User Record"""
```

Comment:
This should not mention deleting an existing user record

Reply:
whoops thanks